schemas/context/opportunity-contactitems.schema.json

```diff
@@ -0,0 +1,48 @@
+{
+  "meta:license": [
+    "Copyright 2019 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/contactitems",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "B2B Opportunity Contact Items",
+  "type": "object",
+  "description": "Defines the list of contact items for a given opportunity",
+  "definitions": {
+    "contactitems": {
+      "properties": {
+        "xdm:roleId": {
+          "title": "Contact Role Id",
+          "description": "Contact role unique identifier.",
+          "type": "string"
+        },
+        "xdm:role": {
+          "title": "Contact Role",
+          "description": "Role of the opportunity contact like finance , sales .",
+          "type": "string"
+        },
+        "xdm:isPrimary": {
+          "title": "Primary",
+          "description": "A flag to signify that this is the primary opportunity contact",
+          "type": "boolean"
+        },
+        "xdm:isDeleted": {
+          "title": "Deleted",
+          "description": "A flag to signify that this opportunity is in deleted state.",
+          "type": "boolean"
+        }
+      }
+    }
+  },
+  "allOf": [
+    {
+      "$ref": "https://ns.adobe.com/xdm/common/extensible#/definitions/@context"
+    },
+    {
+      "$ref": "#/definitions/contactitems"
+    }
+  ],
+  "meta:status": "experimental"
+}
```

Comment:
Can we rename this schema to just roles. This way it can be used to associate roles played by anyone with the context of a function like "Opportunity" or "account". This schema does not talk about contacts as such.

"opportunity-contactmap" schema defines the relationship between a contact and the role.